.github/workflows/jacoco_check.yml

```diff
@@ -21,22 +21,21 @@ on:
     branches: [ master ]
     types: [ opened, edited, synchronize, reopened ]
 
+# The project supports Scala 2.12 and the default version of spark2
+# The CI runs all tests suites for all supported Scala versions at build.yml
+# The codebase for all Scala versions is the same, so the coverage is calculated only once
+# Scala 2.12 is chosen since it is supported by the most wide range of Spark versions and
+# vendor distributions.
+env:
+  scalaLong: 2.12.18
+  scalaShort: "2.12"
+  overall: 80.0
+  changed: 80.0
+
 jobs:
   test:
     name: Build and test
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        include:
-          # The project supports Scala 2.11, 2.12 and the default version of spark2
-          # The CI runs all tests suites for all supported Scala versions at build.yml
-          # The codebase for all Scala versions is the same, so the coverage is calculated only once
-          # Scala 2.12 is chosen since it is supported by the most wide range of Spark versions and
-          # vendor distributions.
-          - scala: 2.12.18
-            scalaShort: "2.12"
-            overall: 80.0
-            changed: 80.0
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
```

Comment:
```suggestion
        uses: actions/checkout@v4.1.1
```

Reply:
I will use `v4` as it is the same as having `v4.1.1` or when they update, having the newest `v4.X.Y`.